content/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst

```diff
@@ -0,0 +1,230 @@
+
+   Starting at Shelf A, the the picker places apples into each package. Next, the picker navigates
+   to Shelf B and places oranges in the packages designated for |SO| 1 and |SO| 3. Finally, the
+   picker pushes the cart to Shelf C and loads packages for |SO| 2 and |SO| 3 with a banana, each.
+
+   With all three |SOS| fulfilled, the picker pushes the cart to the output location, where
+   the packages are sealed and prepared for shipment.
+
+   .. image:: cluster_picking/cluster-example.png
+      :align: center
+      :alt: Show example of fulfilling sales orders 2 and 3 at once.
+
+Configuration
+=============
+
+To enable cluster picking, begin by navigating to :menuselection:`Inventory app --> Configuration
+--> Settings`. Under the :guilabel:`Operations` heading, activate the :guilabel:`Packages` and
+:guilabel:`Batch Transfers` options.
+
+.. image:: cluster_picking/configs.png
+   :align: center
+   :alt: Activate *Packages* and *Batch Transfers* features in the settings.
+
+Since batch picking is used to optimize the *pick* operation in Odoo, the :guilabel:`Storage
+Locations` and :guilabel:`Multi-Step Routes` options under the :guilabel:`Warehouse` heading must
+also be checked on this settings page. When finished, click :guilabel:`Save`.
+
+.. image:: cluster_picking/locations-routes-checkbox.png
+   :align: center
+   :alt: Enable *Storage Locations* and *Multi-Step Routes* Inventory > Configuration > Settings.
+
+.. _inventory/misc/create-package:
+
+Packages setup
+--------------
+
+After the :guilabel:`Packages` feature is enabled, navigate to :menuselection:`Inventory app -->
+Products --> Packages`, and click the :guilabel:`New` button to create a new package.
+
+On the new package form, the :guilabel:`Package Reference` is pre-filled with the next available
+`PACK` number in the system. :guilabel:`Pack Date` is automatically set to the creation date of the
+form.
+
+For the :guilabel:`Package Use` field, the :guilabel:`Disposable Box` radio option should be
+selected if the package is a shipping container. Alternatively, the :guilabel:`Reusable Box` option
+should be selected if the package is simply used as a method of grouping products from the same |SO|
+together before they are moved to the intended shipping container at the output location.
+
+.. seealso::
+   :ref:`Packages <inventory/management/packages>`
+
+.. example::
+   A package intended for cluster picking is named `CLUSTER-PACK-3` for easy identification. For
+   this workflow, the products are directly packed using their intended shipping containers, so
+   :guilabel:`Package Use` is set to :guilabel:`Disposable Box`.
+
+   .. image:: cluster_picking/cluster-package.png
+      :align: center
+      :alt: Create new package form.
+
+Create cluster batch
+====================
+
+To see how cluster picking works in Odoo, navigate to the *Sales* app and create |SOS| that will be
```

Comment:
This should have a gui label since we're telling the reader to click on it.
```suggestion
To see how cluster picking works in Odoo, navigate to the :guilabel:`Sales` app and create |SOS| that will be
```

Reply:
```suggestion
To see how cluster picking works in Odoo, navigate to the :menuselection:`Sales` app and create |SOS| that will be
```